### PR TITLE
Luno update API

### PR DESCRIPTION
**Pull request by DelAnt**
Added missing endpoint send_fee and removed obselete endpoints.

Note, not sure how to add the private endpoints /api/exchange/ especially with the various version options.

### CODE REVIEW

File: dist/cjs/src/luno.js [hunk 1]
```diff
@@ -108,33 +108,35 @@ class luno extends luno$1 {
...
                         'orders/{id}': 1,
-                        'quotes/{id}': 1,
                         'withdrawals': 1,
                         'withdrawals/{id}': 1,
                         'transfers': 1,
+                        // GET /api/exchange/1/move
+                        // GET /api/exchange/1/move/list_moves
+                        // GET /api/exchange/1/candles
+                        // GET /api/exchange/1/transfers
                         // GET /api/exchange/2/listorders
```
Comment (carlosmiei) [inline dist/cjs/src/luno.js:120]: what are these?

Comment (DelAnt) [reply dist/cjs/src/luno.js:120]: New APIs that don't fit into the current structure of the file but I'm not sure how they should. I thought of creating the below API section, but was not sure on the correct convention, now how to handle the version 2 and 3 options in this format. So for now, I'm just adding them commented out. 
```
@@ -77,6 +77,7 @@ export default class luno extends Exchange {
                     'public': 'https://api.luno.com/api',
                     'private': 'https://api.luno.com/api',
                     'exchange': 'https://api.luno.com/api/exchange',
+                    'exchangeprivate': 'https://api.luno.com/api/exchange',
                 },
                 'www': 'https://www.luno.com',
                 'doc': [
@@ -91,6 +92,14 @@ export default class luno extends Exchange {
                         'markets': 1,
                     },
                 },
+                'privateexchange': {
+                    'get': {
+                        'move': 1,
+                        'move/list_moves': 1,
+                        'candles': 1,
+                        'transfers': 1,
+                    }
+                },
                 'public': {
                     'get': {
                         'orderbook': 1,
@@ -1024,7 +1035,7 @@ export default class luno extends Exchange {
         if (Object.keys (query).length) {
             url += '?' + this.urlencode (query);
         }
-        if (api === 'private') {
+        if (api === 'private' || api == 'privateexchange') {
             this.checkRequiredCredentials ();
             const auth = this.stringToBase64 (this.apiKey + ':' + this.secret);
             headers = {
```

Comment (DelAnt) [reply dist/cjs/src/luno.js:120]: Could we make the commit without this section for now. The rest of the changes are good.

Comment (carlosmiei) [reply dist/cjs/src/luno.js:120]: @DelAnt agreed

**Comment by carlosmiei** [comment]
LGTM ✅

File: dist/cjs/src/luno.js [hunk 2]
```diff
@@ -108,33 +108,35 @@ class luno extends luno$1 {
                         'funding_address': 1,
                         'listorders': 1,
                         'listtrades': 1,
+                        'send_fee': 1,
```
Comment (carlosmiei) [inline dist/cjs/src/luno.js:111]: @DelAnt sorry I did not notice you added changes to this file `dist/cjs/src/luno.js`, this is automatically generated, you have to add them to `ts/src/luno.ts` instead

Comment (DelAnt) [reply dist/cjs/src/luno.js]: Sorry. I should have spotted that as well. Does this work now?

Comment (carlosmiei) [reply dist/cjs/src/luno.js]: @DelAnt yeah it's ok now (I restored the other file)

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**